### DISCUSSION TITLE
Snap to Grid option

### DISCUSSION THREAD

**Discussion by cmdrstarkiller** [JulienHeijmans/quicksnap#46] [Ideas]
Hey there! Thanks for making this handy addon! As someone who's transitioning from 3ds max to Blender I really missed this feature a lot before I discovered your addon. I have quick feature suggestion if you don't mind. I think it would be really useful to also have a grid as a "snap to" option similar to how vanilla blender snap has Absolute Grid Snap option

I hope it's not much to ask. Thanks again!

**Comment by JulienHeijmans** [comment]
Thank you for the message, happy to hear people like the addon! 
For the snap to grid option, I plan to do it, but I don't have time to work on QuickSnap at the moment, so it will come but later on.

**Comment by SynrgStudio** [comment]
Hi there @cmdrstarkiller I'm maintaining this addon now in my own repo and I've just addded this feature :D

https://github.com/SynrgStudio/quicksnap/releases

The latest release has the grid snapping with camera aware axis selection :D 

Hope you like the implementation!
